components/member-profile/active-task/index.js

```diff
@@ -1,19 +1,62 @@
 import PropTypes from 'prop-types';
-import classNames from 'components/member-profile/active-task/active-task.module.scss';
-import { timeWas } from 'helper-functions/time-was';
+import classNames from './active-task.module.scss';
+import { timeWas } from '../../../helper-functions/time-was';
+import { percentageofDaysRemaining } from '../../../helper-functions/taskProgress';
+
 const ActiveTask = ({ taskDetails }) => {
-  const { title, purpose, startedOn, endsOn } = taskDetails;
+  const { title, purpose, startedOn, endsOn, percentCompleted } = taskDetails;
   let completedDate = '';
   completedDate = timeWas(startedOn * 1000, false, endsOn * 1000);
+
+  let percentOfTaskLeft = 100 - percentCompleted;
+
+  let daysRemaining = '';
+  daysRemaining = percentageofDaysRemaining(startedOn, endsOn, completedDate);
+
+  function estimatedDaysText() {
+    if (daysRemaining >= percentOfTaskLeft) {
+      return classNames.showTextGreen;
+    } else if (daysRemaining < 50 && percentOfTaskLeft > 75) {
```

Comment:
_Suggestion:_ The way Js uses `if-else` is: 
```
if (condition){
 // Block of Code
} else {
  if (condition) {
    // Block of Code
  } 
}
```

Reply:
> _Suggestion:_ The way Js uses `if-else` is:
> 
> ```
> if (condition){
>  // Block of Code
> } else {
>   if (condition) {
>     // Block of Code
>   } 
> }
> ```

Referred MDN docs

if (condition1)
  statement1
else if (condition2)
  statement2
else if (condition3)
  statement3
...
else
  statementN